kinetic/distribution.yaml

```diff
@@ -0,0 +1,11 @@
+%YAML 1.1
+# ROS distribution file
+# see REP 141: http://ros.org/reps/rep-0141.html
+---
+release_platforms:
```

Comment:
Fixed. I've added 22 and 23, since they are the currently supported releases:

https://fedoraproject.org/wiki/Releases#Current_Supported_Releases